gunicorn_config.py

```diff
@@ -10,6 +10,14 @@
 bind = "0.0.0.0:{}".format(os.getenv("PORT"))
 accesslog = '-'
 
+# See AWS doc
+# > We also recommend that you configure the idle timeout of your application
+# to be larger than the idle timeout configured for the load balancer.
+# > By default, Elastic Load Balancing sets the idle timeout value for your load balancer to 60 seconds.
+# https://docs.aws.amazon.com/elasticloadbalancing/latest/application/application-load-balancers.html#connection-idle-timeout
+if os.environ.get("NOTIFY_ENVIRONMENT", "") == "production":
```

Comment:
This env is set to this value in staging as well so we will be able to test it beforehand